### DISCUSSION TITLE
Phoenix MST7 timezone seems to have daylight savings enabled

### DISCUSSION THREAD

**Discussion by caykroyd1** [roleoroleo/yi-hack-Allwinner-v2#896] [Q&A]
I think it is actually MST7 with daylight savings.  so it is currently 1 hour off.  Of course not a big deal in the scheme of things, but since the Yi Home 2k Pro have native OSD, I would like to see this fixed.

**Comment by roleoroleo** [comment]
Please, try with MST (without 7).

**Reply by caykroyd1** [reply]
It seemed after a couple reboots and maybe toggles of some other features, MST7 works.  Not sure exactly what happened but it is fixed now.
